tools/ci/tag_master.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+import os
+import sys
+import urllib2
+
+here = os.path.abspath(os.path.dirname(__file__))
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+if not(wpt_root) in sys.path:
```

Comment:
This is pretty weird. `not(str)` is always `False`, and `False` will never appear in `sys.path`.